dor/cli/upload.py

```diff
@@ -83,16 +83,15 @@ async def _run_upload(
     for exception in exceptions:
         if isinstance(exception, UploadError):
             typer.secho(f"Upload error: {exception}", fg="white", bg="red", bold=True)
-            typer.echo(f"Details: {exception.message}, Code: {exception.code}", err=True)
+            typer.echo(f"Details: {exception.message}, Code: {exception.code}")
```

Comment:
Why would errors not go to STDERR?

Reply:
STDERR goes to the terminal. It's always unbuffered so they'll see the errors sooner than STDOUT output (assuming we haven't configured things to flush STDOUT immediately).

Reply:
Okay, I'll change it back.

Reply:
Apologies for my ignorance, haven't worked with `stderr` much